src/connector/null-connector.ts

```diff
@@ -20,14 +22,14 @@ export class NullConnector extends Connector<NullPublicChannel, NullPrivateChann
     /**
      * Listen for an event on a channel instance.
      */
-    listen(name: string, event: string, callback: Function): NullChannel<NullChannels> {
+    listen(name: string, event: string, callback: Function): AnyNullChannel {
```

Comment:
Since this specifically returns `NullChannel` would we want the return type to be that instead of `AnyNullChannel`?

Reply:
Yeah, makes sense to be more specific. I can get rid of `AnyNullChannel` altogether then. 👍🏼